pages/Configuring/Using-hyprctl.md

```diff
@@ -91,28 +93,42 @@ splash - prints the current random splash
 getoption [option] - gets the config option status (values)
 ```
 
-# Batch
+For the `getoption` command, you need to pass the command in the following format:
```

Comment:
*For the `getoption` command, the option name should be in the following format:*
*section:option*